sdk/monitor/monitor-query/README.md

```diff
@@ -51,40 +51,40 @@ Using [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/blob/ma
 
 ```javascript
 const { DefaultAzureCredential } = require("@azure/identity");
-const { LogsClient, MetricsClient } = require("@azure/monitor-query");
+const { LogsQueryClient, MetricsQueryClient } = require("@azure/monitor-query");
 
 const credential = new DefaultAzureCredential();
 
-const logsClient = new LogsClient(credential);
+const logsQueryClient = new LogsQueryClient(credential);
 // or
-const metricsClient = new MetricsClient(credential);
+const metricsQueryClient = new MetricsQueryClient(credential);
 ```
 
 More information about `@azure/identity` can be found [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/README.md)
 
 ## Key concepts
 
-The [`LogsClient`]<!--[msdocs_logs_client]--> allows you to query logs, using the [Kusto query language][kusto_query_language]. This data can be queried in the
+The [`LogsQueryClient`]<!--[msdocs_logs_client]--> allows you to query logs, using the [Kusto query language][kusto_query_language]. This data can be queried in the
 portal using tables like `AppEvents`, `AppDependencies` and others.
 
-The [`MetricsClient`]<!--[msdocs_metrics_client]--> allows you to query metrics.
+The [`MetricsQueryClient`]<!--[msdocs_metrics_client]--> allows you to query metrics.
 
 ## Examples
 
 #### Querying logs
 
-The LogsClient can be used to query a Monitor workspace using the Kusto Query language.
+The `LogsQueryClient` can be used to query a Monitor workspace using the Kusto Query language.
 
 ```javascript
-const { LogsClient } = require("@azure/monitor-query");
+const { LogsQueryClient } = require("@azure/monitor-query");
 const { DefaultAzureCredential } = require("@azure/identity");
 
 const azureLogAnalyticsWorkspaceId = "<the Workspace Id for your Azure Log Analytics resource>";
-const logsClient = new LogsClient(new DefaultAzureCredential());
+const logsQueryClient = new LogsQueryClient(new DefaultAzureCredential());
 
 async function run() {
   const kustoQuery = "AppEvents | limit 1";
-  const result = logsClient.queryLogs(azureLogAnalyticsWorkspaceId, kustoQuery);
+  const result = logsQueryClient.queryLogs(azureLogAnalyticsWorkspaceId, kustoQuery);
```

Comment:
Assuming `queryLogs` is async, should we await in this snippet?

Reply:
Fixed!